azure-ts-appservice/README.md

```diff
@@ -0,0 +1,62 @@
+# Azure App Service with SQL Database and Application Insights
+
+Starting point for building web application hosted in Azure App Service.
+
+Provisions Azure SQL Database and Azure Application Insights to be used in combination
+with App Service.
+
+## Running the App
+
+1.  Create a new stack:
+
+    ```
+    $ pulumi stack init azure-appservice
+    ```
+
+1.  Login to Azure CLI (you will be prompted to do this during deployment if you forget this step):
+
+    ```
+    $ az login
+    ```
+
+1.  Restore NPM dependencies:
+
+    ```
+    $ npm install
+    ```
+
+1.  Build the Typescript project:
+
+    ```
+    $ npm run build
+    ```
+
+1. Define SQL Server password (make it complex enough to satisfy Azure policy):
+
+    ```
+    pulumi config set azpulumi:sqlPassword <value>
```

Comment:
If you do take the suggestion below to replace `azpulumi` with `pulumi.getStack()`, then this can become just `pulumi config set sqlPassword <value>`.

Reply:
Hmm... `getStack()` returns stack name e.g. `mystack` while default config is saved under app name e.g. `azure-appservice`